src/codegen/stub_generator.py

```diff
@@ -10,14 +9,15 @@
 
 STUBS_NAMESPACE = "nidaqmx._stubs"
 PROTO_PARENT_NAMESPACES = ["src.codegen", "protos"]
-STUBS_PATH = pathlib.Path(__file__).parent.parent.parent / "generated" / STUBS_NAMESPACE.replace(".", "/")
+STUBS_PATH = (
+    pathlib.Path(__file__).parent.parent.parent / "generated" / STUBS_NAMESPACE.replace(".", "/")
+)
 PROTO_PATH = pathlib.Path(__file__).parent.parent.parent / "src" / "codegen" / "protos"
 PROTO_FILES = list(PROTO_PATH.rglob("*.proto"))
 
 
 def generate_stubs():
     """Generate and fixup gRPC Python stubs."""
-    remove_generated_files(STUBS_PATH, PROTO_PATH)
```

Comment:
Why delete this? Is it because [the PR workflow](https://github.com/ni/nidaqmx-python/blob/master/.github/workflows/PR.yml) already deletes generated/nidaqmx?